docs/Catalog/SearchQuery.md

```diff
@@ -1,44 +1,46 @@
 <!-- markdownlint-disable -->
-Quilt provides support for queries in the ElasticSearch DSL, as
+Quilt provides support for queries in the Elasticsearch DSL, as
 well as SQL queries in Athena.
 
-## ElasticSearch
+## Elasticsearch
 
 The objects in Amazon S3 buckets connected to Quilt are synchronized to
-an ElasticSearch cluster, which provides Quilt's search features.
-This is available directly in the navigation bar, or 
-for custom queries you can use the **Queries** tab in the Quilt catalog
-to directly query the ElasticSearch cluster.
+an Elasticsearch cluster, which provides Quilt's search features.
 
-Quilt uses ElasticSearch 6.7
+Quilt uses Elasticsearch 6.7
 ([docs](https://www.elastic.co/guide/en/elasticsearch/reference/6.7/index.html)).
 
 ### Indexing
 Quilt maintains a near-realtime index of the objects in your S3
-bucket in ElasticSearch.  Each bucket corresponds to one or more
-ElasticSearch indexes. As objects are mutated in S3, Quilt uses an
-event-driven system (via SNS and SQS) to update ElasticSearch.
+bucket in Elasticsearch.  Each bucket corresponds to one or more
+Elasticsearch indexes. As objects are mutated in S3, Quilt uses an
+event-driven system (via SNS and SQS) to update Elasticsearch.
 
 There are two types of indexing in Quilt:
 * *shallow* indexing includes object metadata (such as the file name and size)
 * *deep* indexing includes object contents. Quilt supports deep
 indexing for the following file extensions:
+  * .csv, .html, .json, .md, .rmd, .rst, .tab, .txt, .tsv (plain-text formats)
```

Comment:
We don't support preview of `.rst` https://open.quiltdata.com/b/quilt-example/tree/dima/node_modules2/highlight.js/docs/language-requests.rst

Reply:
There are tasks for `.rst` for both frontend and backend, and from my comments there I see, that main job should be done on backend side

Reply:
Ok, this is about indexing not preview